dist/source/wrapper/DdUrlTransfer.brs

```diff
@@ -769,24 +774,69 @@ function getTracedHeaderType(url as string, tracingHeaderTypes as object) as dyn
 end function
 
 ' ----------------------------------------------------------------
-' Generates a unique identifier compatible with Datadog's APM trace and span IDs
+' Generates a unique identifier as a 64 bits random number
+' @param radix (integer) the radix to use when representing the id as a string
+' @return (string) the generated Unique id
+' ----------------------------------------------------------------
+function generateUniqueId64(radix = 10 as integer) as string
+    maxInt = 4294967295
+    return printIdToString(0, 0, Rnd(maxInt) - 1, Rnd(maxInt) - 1, radix)
+end function
+
+' ----------------------------------------------------------------
+' Generates a unique identifier as a 128 bits random number
+' @param radix (integer) the radix to use when representing the id as a string
 ' @return (string) the generated Unique id
 ' ----------------------------------------------------------------
-function generateUniqueId(radix = 10 as integer) as string
+function generateUniqueId128(radix = 10 as integer) as string
     maxInt = 4294967295
-    low& = Rnd(maxInt) - 1
-    high& = Rnd(maxInt) - 1
+    return printIdToString(Rnd(maxInt) - 1, Rnd(maxInt) - 1, Rnd(maxInt) - 1, Rnd(maxInt) - 1, radix)
+end function
+
+' ----------------------------------------------------------------
+' Generates a unique identifier as a 128 bits random number, fit for Datadog's specification
+' @return (array) the array with the generated id as a list of strings:
+'     - the full 128 bit id in hexadecimal
+'     - the low 64 bit part of the id in decimal (compatible with Datadog's legacy x-datadog-trace-id header)
+'     - the high 64 bit part of the id in id in hexadecimal
+' ----------------------------------------------------------------
+function generateUniqueIdDd() as object
+    maxInt = 4294967295
```

Comment:
I'm guessing there's not good way of making this more global instead of having it repeated in each function?